templates/primary-secondary/primary-secondary.js

```diff
@@ -52,8 +57,17 @@ class TemplatePrimarySecondary extends LitElement {
 	}
 
```

Comment:
I'm wondering how the width-type should affect the footer? What does design say about that? Like could we constrain the buttons and the box-shadow somehow automatically?

Reply:
Spoke with design again, after a bit more discussion, they would like the footer to be the full-width, but the content inside the footer should be constrained to 1230px. So I'll be updating the current PR to do that!

Reply:
Awesome, that's what I was hoping for 👍